lib/mayaUsd/ufe/UsdUndoAttributesCommands.cpp

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright 2022 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include "UsdUndoAttributesCommands.h"
+
+#include <mayaUsd/ufe/UsdAttributes.h>
+
+#include <ufe/hierarchy.h>
+
+PXR_NAMESPACE_USING_DIRECTIVE
```

Comment:
Do you really need this? According to our coding standards we should use the fully qualified names if we can.

Reply:
Since no USD classes are mentioned in that file, it is definitely not needed.